bf2fc6cc711aee1a0c2a/kas-fleet-manager
# test: Increase unit test coverage in internal/kafka/internal/services

## Description
Increase unit test code coverage in internal/kafka/internal/services where possible or ignore files, for which writing unit tests is not feasible in the make test make target.

Jira: [Here](https://issues.redhat.com/browse/MGDSTRM-7980)

### Before
![Before](https://user-images.githubusercontent.com/59924001/165798179-78b8b8bf-2283-4f84-b08d-a6272a168e18.png)
### After
![After](https://user-images.githubusercontent.com/59924001/165798235-67d4fbcd-a395-408a-958c-6581eb337367.png)

## Verification Steps
1. Run make test.
2. Run make test/html/coverage/report.
3. Should be able to see the improved test coverage.

## Checklist (Definition of Done)
